Resources/ALC236/Info.plist

```diff
@@ -108,6 +108,14 @@
 				<key>Path</key>
 				<string>layout54.xml.zlib</string>
 			</dict>
+			<dict>
```

Comment:
Could you please reorder it in ascending order by layout-id? Same for platform-id below.

Reply:
Sorry, I didn't understand the question. All files are in the following order. 54-77(mine)-99.